.github/workflows/pull-request.yml

```
@@ -251,21 +251,28 @@
 
-      - name: Merge Coverage Reports
-        if: steps.changed-files.outputs.only_changed != 'true'
-        run: |
-          mkdir -p coverage
-          if ! npx lcov-result-merger 'coverage/*/lcov.info' > 'coverage/lcov.info'; then
-            echo "Failed to merge coverage reports"
-            exit 1
-          fi     
+      - name: Upload Vitest Coverage to Codecov
+        if: steps.changed-files.outputs.any_changed == 'true'
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)